@AttilaVamos the check failure seems unrelated:
"CMake Error at cmake_modules/commonSetup.cmake:756 (MESSAGE):
-- Found Bison v3.5.1
  You need Gnu c++ version 7.3.0 or later to build this project (version
-- Configuring incomplete, errors occurred!"

--- a/dali/sasha/saserver.cpp
+++ b/dali/sasha/saserver.cpp
@@ -298,8 +298,6 @@ static constexpr const char * defaultYaml = R"!!(
 version: 1.0
 sasha:
   name: sasha
-  logging:
-    detail: 100
 )!!";
 
 int main(int argc, const char* argv[])

--- a/dali/server/daserver.cpp
+++ b/dali/server/daserver.cpp
@@ -398,8 +398,6 @@ version: 1.0
 dali:
   name: dali
   dataPath: "/var/lib/HPCCSystems/dalistorage"
-  logging:
-    detail: 100
 )!!";
 
 
@@ -652,7 +650,7 @@ int main(int argc, const char* argv[])
         Owned<IMPServer> mpServer = getMPServer();
         Owned<IAllowListHandler> allowListHandler = createAllowListHandler(populateAllowListFromEnvironment, formatDaliRole);
         mpServer->installAllowListCallback(allowListHandler);
-        setMsgLevel(fileMsgHandler, serverConfig->getPropInt("SDS/@msgLevel", 100));
+        setMsgLevel(fileMsgHandler, serverConfig->getPropInt("SDS/@msgLevel", DebugMsgThreshold));
 #endif
         startLogMsgChildReceiver();
         startLogMsgParentReceiver();

--- a/helm/examples/logging/README.md
+++ b/helm/examples/logging/README.md
@@ -46,7 +46,7 @@ The available target categories include disaster(DIS), error(ERR), warning(WRN),
     helm install myhpcc ./hpcc --set dali[0].logging.classes="ALL-PRO" --set dali[0].name="mydali"
 
 ### Log Detail Level Configuration
-Log output verbosity can be adjusted from "critical messages only" (1) up to "report all messages" (100). By default, the log level is set highest (100). 
+Log output verbosity can be adjusted from "critical messages only" (1) up to "report all messages" (100). The default log level is rather high (80) and should be adjusted accordingly.
 
     For example, verbosity should be medium for all components:
     helm install myhpcc ./hpcc --set global.logging.detail="50"

--- a/helm/hpcc/values.yaml
+++ b/helm/hpcc/values.yaml
@@ -13,7 +13,7 @@ global:
 
   # logging sets the default logging information for all components. Can be overridden locally
   logging:
-    detail: 100
+    detail: 80
 
   # Specify a defaultEsp to control which eclservices service is returned from Std.File.GetEspURL, and other uses
   # If not specified, the first esp component that exposes eclservices application is assumed.

--- a/roxie/roxie/roxie.cpp
+++ b/roxie/roxie/roxie.cpp
@@ -51,8 +51,6 @@ roxie:
   localSlave: true
   numChannels: 1
   queueNames: roxie.roxie
-  logging:
-    detail: 100
 )!!";
 
 int main(int argc, const char *argv[])

--- a/system/jlib/jlog.hpp
+++ b/system/jlib/jlog.hpp
@@ -239,7 +239,7 @@ inline unsigned LogMsgClassFromAbbrev(char const * abbrev)
 }
 
 typedef unsigned LogMsgDetail;
-#define DefaultDetail   100
+#define DefaultDetail   DebugMsgThreshold
 #define TopDetail (LogMsgDetail)-1
 
 /*

